bin/bank/pycbc_brute_bank

```diff
@@ -337,6 +371,14 @@ gen = GenUniformWaveform(args.buffer_length,
     args.sample_rate, args.low_frequency_cutoff)
 bank = TriangleBank()
 
+if args.num_cores > 1:
+    def wf_wrapper(p):
+        try:
+            hp = gen.generate(**p)
+            return hp
+        except Exception:
+            return numpy.nan
```

Comment:
I would return `None` and not `numpy.nan`

Reply:
@raffienficiaud changed to None